src/components/Home.js

```diff
@@ -0,0 +1,113 @@
+// import { FaArrowAltCircleRight } from 'react-icons/fa';
+import styles from '../CSS/Home.module.css';
+
+const Home = () => {
+  const { news, categories, totalResults } = useSelector((state) => state.news);
+  const [selectedItem, setSelectedItem] = useState(null);
+  const [filter, setFilter] = useState('');
+
+  const handleCardClick = (item) => {
+    setSelectedItem(item);
+  };
+
+  const handleFilterChange = (event) => {
+    setFilter(event.target.value);
+  };
+
+  const filterNews = news.filter((item) => item.title.toLowerCase().includes(filter.toLowerCase()));
+  return (
+    <div className={styles.homeCont} data-testid="home">
+      <div className={styles.headCont}>
+        <h1 className={styles.titleN}>News</h1>
+        <div>
+          <p className={styles.total}>
+            Total Results :
+          </p>
+          <p>{totalResults}</p>
+          <input
+            type="text"
+            value={filter}
+            onChange={handleFilterChange}
+            placeholder="Filter news..."
+            className={styles.filterInput}
+          />
+        </div>
+      </div>
+      <div className={styles.homeCard}>
+        {filterNews.map((item) => (
+          <div className={styles.card} key={item.title} data-testid="card">
+            <Link
+              to={`/detail/${encodeURIComponent(item.title)}`}
+              className={styles.single_card}
+            >
+              <div
+                onClick={() => handleCardClick(item)}
+                onKeyDown={() => handleCardClick(item)}
+                role="link"
+                tabIndex={0}
+                className={styles.card_content}
+              >
+                <div className={styles.arrow_wrapper}>
+                  <span className={styles.arrow_icon}>
+                    <svg
+                      width="24"
+                      height="24"
+                      xmlns="http://www.w3.org/2000/svg"
+                      viewBox="0 0 26 26"
+                      fillRule="evenodd"
+                      clipRule="evenodd"
+                    >
+                      <path d="M12 0c-6.623 0-12 5.377-12 12s5.377 12 12 12 12-5.377 12-12-5.377-12-12-12zm0 1c-6.071 0-11 4.929-11 11s4.929 11 11 11 11-4.929 11-11-4.929-11-11-11zm4.828 11.5l-4.608 3.763.679.737 6.101-5-6.112-5-.666.753 4.604 3.747h-11.826v1h11.828z" fill="white" />
+                    </svg>
+                  </span>
+                </div>
+                <img
+                  src={item.image}
+                  alt={item.title}
+                  className={styles.card_image}
+                />
+                <p className={styles.titlecd}>{item.title}</p>
+                <p className={styles.titlecd}>
+                  <p className={styles.total}>Results:</p>
+                  <br />
+                  {' '}
+                  {categories[item.source.name]}
+                </p>
+              </div>
+            </Link>
+          </div>
+        ))}
+      </div>
```

Comment:
- [ ] Great job in implementing some changes requested by previous reviewer. At the moment when I compare your design to the given design, there are still some very visible differences.
- You have a space to the left and right of the cards, but on the given design the cards occupy the entire screen. Check screenshots below:
**Your design**
![image](https://github.com/Christelle-12/react-redux-capstone/assets/62849872/d8793c4e-21f3-45e5-9dda-da5d621871c1)
**Given design**
![image](https://github.com/Christelle-12/react-redux-capstone/assets/62849872/7e909be3-12e4-4f4f-b829-ff9a2e9dfb7b)
As you can clearly see from the given design above, the cards occupy the entire screen. There is no gab to the left and right. Kindly fix your cards to respect the given design.
- Secondly, at the top of your design where you have `News` and `Total Results: 3901`, is suppose to be two cards as per the given design. I suggest that you make that top part 2 cards with the right card having News and Total Results: 3901 and the left card having the filter. In that way your design will look similar to the given design. See screenshots below for clarity.
**Your design**
![image](https://github.com/Christelle-12/react-redux-capstone/assets/62849872/b2813223-80f5-4a64-96e2-3bacfe89e8d0)
**Given design**
![image](https://github.com/Christelle-12/react-redux-capstone/assets/62849872/ff8ce109-68c5-4109-99e7-b6490b601c08)

Reply:
As the previous reviewer mentioned to stick as much possible to the design you can put total rasults in the right side with News use an image or globe map  something referring to international news the in the second row before the list you can put the search input